tests/ObjectCreationTests.fs

```diff
@@ -0,0 +1,51 @@
+﻿module ObjectCreationTests
+
+
+open Expecto
+open System.Text.Json
+open System.Text.Json.Nodes
+// These tests use the objects in TestObjects and compare them to the objects parsed from ReferenceObjects
+// a `DeepEquals` method is curtrently not available for System.text.Json, so we have to either wait or write such a function on our own.
+// i'll mark these tests as pending for now.
+// see also: https://stackoverflow.com/questions/60580743/what-is-equivalent-in-jtoken-deepequals-in-system-text-json
+
+[<PTests>]
```

Comment:
Why `PTests`?

Reply:
https://github.com/haf/expecto#pending-tests

The tests are there, and get discovered by test explorer, but not run. Instead, they will be marked as `skipped` during execution. Together with the comment it should be clear what to do, once `JsonObject` has a good equality method (these tests will fail with normal/sequenceEquality, but we absolutely want object comparison sometimes in the future, because for string comparison the order of properties matters, while objects with the same properties in different orders should be considered equal)

Reply:
Whoops didn't notice the comment 😅
Thanks for the explanation, expected something along these lines an will merge now